projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss

```diff
@@ -23,15 +23,15 @@
 
 .nav-link-text {
   margin-top: 0.125rem;
-  white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
   max-width: calc(100% - #{spacing.$spacing-02});
-  @include type.type-style('heading-03');
+  @include type.type-style('label-01');
```

Comment:
Looking through this again and `label-01` seems too small for me, @icrc-jofrancisco:

![CleanShot 2024-03-28 at 1  01 19@2x](https://github.com/openmrs/openmrs-ngx-formentry/assets/8509731/38138baf-cd92-40af-b661-33601e377dcf)

Reply:
How about `body-01`?

![CleanShot 2024-03-28 at 1  10 45@2x](https://github.com/openmrs/openmrs-ngx-formentry/assets/8509731/4d5c1e75-3048-4dba-a042-7cfde08fcd63)

Reply:
Agreed. Thanks.